File: common/src/main/java/com/thesilentnights/commands/common/CommonCommands.java
package com.thesilentnights.commands.common;

import com.mojang.brigadier.CommandDispatcher;
import com.mojang.brigadier.builder.LiteralArgumentBuilder;
import net.minecraft.commands.CommandSourceStack;


public interface CommonCommands {
    static void registerCommands(CommandDispatcher<CommandSourceStack> commandDispatcher) {
        commandDispatcher.register(new LoginCommands().getCommand());
        commandDispatcher.register(new RegistrarCommands().getCommand());
        commandDispatcher.register(new LogoutCommand().getCommand());
    }

    LiteralArgumentBuilder<CommandSourceStack> getCommand();
}

File: src/main/kotlin/com/thesilentnights/easylogin/commands/common/CommonCommands.kt
package com.thesilentnights.easylogin.commands.common

import com.mojang.brigadier.builder.LiteralArgumentBuilder
import net.minecraft.commands.CommandSourceStack
import org.koin.core.component.KoinComponent

interface CommonCommands: KoinComponent {
    val command: LiteralArgumentBuilder<CommandSourceStack>
}
